addon-templates/kubectl/dashboard/dashboard-controller.yaml

```diff
@@ -16,20 +17,19 @@ spec:
         k8s-app: kubernetes-dashboard
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
```

Comment:
marking the addons as critical is important and to make sure its always scheduled.